CHANGELOG.md

```diff
@@ -5,12 +5,40 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+<!-- NOTE(eddyb) sections from the original template:
+
+### Added
+- New features go here in a bullet list
+
+### Changed
+- Changes to existing functionality go here in a bullet list
+
+### Deprecated
+- Mark features soon-to-be removed in a bullet list
+
+### Removed
+- Features that have been removed in a bullet list
+
+### Fixed
+- Bug fixes in a bullet list
+
+### Security
```

Comment:
```suggestion
### Added ⭐
- New features go here in a bullet list

### Changed 🛠
- Changes to existing functionality go here in a bullet list

### Deprecated 🚧
- Mark features soon-to-be removed in a bullet list

### Removed 🔥
- Features that have been removed in a bullet list

### Fixed 🩹
- Bug fixes in a bullet list

### Security 🔐
```

Sorry, I like my emojis 😁

Reply:
Heh, thanks! (I just didn't know where to get the emoji versions from)